### PR TITLE
Add viewTasks and DeleteTask commands

### PR DESCRIPTION
**Pull request by rachelljt**
added new commands - viewTasks and DeleteTask

### CODE REVIEW

File: src/main/java/seedu/address/logic/commands/ViewTasksCommand.java [hunk 1]
```diff
@@ -0,0 +1,24 @@
...
+public class ViewTasksCommand extends Command {
+
+    public static final String COMMAND_WORD = "viewTasks";
+
+    public static final String MESSAGE_SUCCESS = "Listed all Tasks";
+
+    @Override
+    public CommandResult execute(Model model) throws CommandException {
+        requireNonNull(model);
+        model.updateFilteredPersonList(PREDICATE_DO_NOT_SHOW_ALL_PERSONS);
```
Comment (zatkiller) [inline src/main/java/seedu/address/logic/commands/ViewTasksCommand.java]: Are we supposed to update the person list?

**Comment by zatkiller** [comment]
Just have 1 small thing to check on, which is the updating of updatedPersonList when we are updating tasks. The rest LGTM

File: src/main/java/seedu/address/logic/commands/ViewTasksCommand.java (hunk 1 above)
Comment (rachelljt) [reply src/main/java/seedu/address/logic/commands/ViewTasksCommand.java]: ohhh okay can i will make the changes

File: src/main/java/seedu/address/logic/commands/DeleteTaskCommand.java [hunk 2]
```diff
@@ -0,0 +1,53 @@
...
+import java.util.List;
+
+import seedu.address.commons.core.Messages;
+import seedu.address.commons.core.index.Index;
+import seedu.address.logic.commands.exceptions.CommandException;
+import seedu.address.model.Model;
+import seedu.address.model.task.Task;
+
+/**
+ * Deletes a person identified using it's displayed index from the address book.
```
Comment (zatkiller) [inline src/main/java/seedu/address/logic/commands/DeleteTaskCommand.java]: Also might need to change the comment here, other than this small issue can merge

**Review by zatkiller — APPROVED** [review]
LGTM. Just remember to change the comment

File: src/main/java/seedu/address/logic/commands/DeleteTaskCommand.java (hunk 2 above)
Comment (rachelljt) [reply src/main/java/seedu/address/logic/commands/DeleteTaskCommand.java]: okay!

**Pull request merged**